### ISSUE TITLE
Attempting to import reference database into QIIME2

### ISSUE THREAD

**Issue by bkramer88** [moyn413/nifHdada2#3]
Hi! I'm trying to use your nifH reference fasta file (nifH_dada2_phylum_v1.1.0.fasta) into QIIME 2 to taxonomically classify my own samples. However, the file does not successfully import, mainly because some of the sequences are identical to one another. I tried to add unique IDs for each sequence following the lowest taxonomic level, but that didn't work either.

Is there a version of the fasta file where the sequence IDs are included? Moreover, do you have a corresponding taxonomy file to go with the reference database? QIIME has specific import functions for the reference sequence file and the taxonomy file to build the actual classifier.

Thanks in advance for all your help!

Ben

**Comment by moyn413** [comment]
Hi Ben, 
Thanks for your message! I can definitely generate a fasta file with unique IDs. Is there a particular way you want the taxonomy file formatted? Maybe we can chat more via email or zoom. I would be happy to discuss more in detail what you need, and then hopefully I can also add a QIIME 2 formatted file to same page, which would be a great resource for the community.

**Comment by bkramer88** [comment]
Hi!

Sorry for the delayed response. I wanted to be able to give you an update on my work progress in trying to figure this out...turns out I did!

I was able to successfully import the fasta file along with a taxonomy file that included the original taxonomic information provided in your original fasta file in github...or I should say the postdoc in my advisor's lab - Dr. Deepak Nanjappa, was able to do that in bash.

Specifically, he extracted the taxonomic information for each sequence into a new txt file then gave each reference sequence their own ID, stating with 1001, 1002, 1003, etc.

He was also able to align, trim, and degap the sequences in MAFFT using 15 of my representative sequences that were already denoised in Q2 using dada2.

I'm happy to chat further about this if you would like! My work email is benjamin.j.kramer@stonybrook.edu

Also, I don't know if you know this, but does the Zehr lab update their database regularly with new sequences/taxonomic information? If so, I would like to try and include the most updated taxonomic information possible in characterizing my sequences.

Thanks and hope to hear from you soon!

Ben
